kiali-operator/templates/deployment.yaml

```diff
@@ -19,6 +19,10 @@ spec:
         # required for the operator SDK metric service selector
         name: {{ include "kiali-operator.fullname" . }}
       {{- include "kiali-operator.labels" . | nindent 8 }}
+      {{- include "kiali-server.labels" . | nindent 8 }}
```

Comment:
this should not be here. the operator is unrelated to the kiali-server (in fact, I don't even think kiali-server.labels is even defined in the operator helm chart).

Reply:
Fixed in 901bfdc2165055f7e2b4cbf731cf29ec12ce2e11. Sorry, don't know how this slipped through.